techniques/html/H99.html

```diff
@@ -117,12 +117,12 @@ <h3>Example 1: Page list in <abbr title="HyperText Markup Language">HTML</abbr>
    …
  &lt;/ol&gt;
 &lt;/nav&gt;</code></pre>
-				<p>The destinations of the page list links are usually explicit page break markers like the following:</p>
+				<p>ページリストリンクのリンク先は通常、次のような明示的なページ区切りマーカーである:</p>
 				<pre xml:space="preserve"><code class="language-html">&lt;span id="pg1" role="doc-pagebreak" aria-label="page 1"&gt;</code></pre>
 				</section>
 				<section class="example" id="example-2-page-list-in-epub-3-publications">
-				<h3>Example 2: Page list in EPUB 3 publications</h3>
-				<p>EPUB 3 uses the <code>epub:type</code> attribute to identify the page list in the navigation document. The attribute is used on a nav element with the value "page-list" (note the different hyphenation). The <abbr title="Accessible Rich Internet Applications">ARIA</abbr> <code class="language-html">role</code> attribute has no effect unless the navigation document can be directly accessed by users (i.e., the document is in the EPUB spine). Like the previous HTML example, the contents of the <code class="language-html">nav</code> element is a list of links to the page break locations.</p>
+				<h3>事例 2: EPUB 3 出版物のページリスト</h3>
+				<p>EPUB 3では、ナビゲーションドキュメント内のページリストを識別するために <code>epub:type</code> 属性を使用する。この属性は、値が "page-list" である nav 要素で使用される (ハイフネーションが異なることに注意する)。<abbr title="Accessible Rich Internet Applications">ARIA</abbr> <code class="language-html">role</code> 属性は、利用者がナビゲーションドキュメントに直接アクセスできる場合 (つまり、ドキュメントが EPUB の spine 内にある場合) を除き、効果がない。前の HTML の例と同様に、<code class="language-html">nav</code> 要素の内容は改ページ位置へのリンクのリストである。</p>
```

Comment:
「値が "page-list" である nav 要素」というのは変ですね。nav要素にepub:type="page-list"を指定するという話かと思います (参考: https://www.w3.org/TR/epub-a11y-tech-111/#example-a-page-list-nav-element )。

それからここですが、

>利用者がナビゲーションドキュメントに直接アクセスできる場合 (つまり、ドキュメントが EPUB の spine 内にある場合) を除き、効果がない。

これはおそらく解釈が逆のように思います。EPUBのspine というのは `<spine><itemref idref="cover"/><itemref idref="chapter1"/></spine>` のようにドキュメントのIDを列挙するタイプのデータで、UAがこれを解釈して目次を生成します。このような spine 要素に role 属性をつけたりしても効果がない、という話をしているのだと思います。

こんな感じかなと。

```suggestion
				<p>EPUB 3では、ナビゲーションドキュメント内のページリストを識別するために <code>epub:type</code> 属性を使用する。この属性の値を "page-list" とし (ハイフネーションが異なることに注意)、nav 要素において使用する。<abbr title="Accessible Rich Internet Applications">ARIA</abbr> <code class="language-html">role</code> 属性は、利用者がナビゲーションのドキュメントに直接アクセスできない場合には、効果がない (つまり、そのドキュメントが EPUB の spine 内にある場合)。前の HTML の例と同様に、<code class="language-html">nav</code> 要素の内容は改ページ位置へのリンクのリストである。</p>
```